### DISCUSSION TITLE
Is there a way to mute logger output while testing?

### DISCUSSION THREAD

**Discussion by SkifAlef** [adonisjs/core#3921] [Help] [answered]
I'd like to mute server logs via exceptions logger when I run test.

I mean I'd like to get only passed||failed messages from test
![image](https://user-images.githubusercontent.com/679283/192137437-2b5797d7-59a8-437f-966b-9e422c7bfa12.png)

Is there a simple way to achieve this without implementing custom exceptions with report method everywhere?

**Comment by thetutlage** [comment, marked as answer]
You can implement the `report` method within the global exception handler and disable it when in testing environment.

```ts
import Application from '@ioc:Adonis/Core/Application'

public async report(error: any, ctx: HttpContextContract) {
  if (process.env.NODE_ENV === 'test') {
    return
  }

  return super.report(error, ctx)
}
```
